From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Date: Sat, 22 Aug 2026 12:00:00 +0200
Subject: [PATCH] gtk: Skip header files that cannot be found

mkorder.awk ignores preprocessor conditionals while chasing #include
lines, so it looks for glib/gwin32.h, which glib only installs on Windows.
Skip headers that are not there instead of failing the build.

Upstream-Status: Pending
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 packages/gtk/mkorder.awk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/gtk/mkorder.awk b/packages/gtk/mkorder.awk
index ea17c0dc906e..ecb0eb2a9ce0 100644
--- a/packages/gtk/mkorder.awk
+++ b/packages/gtk/mkorder.awk
@@ -64,7 +64,7 @@ function process_file(name, file) {
 	return
     processed[file] = 1
     if (file == "")
-	print "error: cannot find " name
+	return
     else {
 	scan(file)
         print file
